tslint.json

```diff
@@ -61,6 +63,7 @@
       "check-else",
       "check-whitespace"
     ],
+    "prefer-const": false,
```

Comment:
Should set this back to `true` at some point.  But probably in its own PR, as it flags a lot of stuff.